RELEASE-NOTES.md

```diff
@@ -33,6 +33,19 @@
 * Fixed problem with whitespaces in `snow connection add` command
 * Added check for the correctness of token file and private key paths when addind a connection
 
+# v3.0.0
+
+# Unreleased version
+## Backward incompatibility
+
+## Deprecations
+* `snow spcs job` command is removed. This command is a deprecated command which is now replaced by the new command `snow spcs service execute-job`
```

Comment:
Taking into account that `snow spcs job` was hidden (officially unreleased) I don't think we need to add this note.

Reply:
Make sense. Removed